src/themes/default/layouts/Default.vue

```diff
@@ -60,16 +60,21 @@ export default {
       isSidebarOpen: state => state.ui.sidebar,
       isMicrocartOpen: state => state.ui.microcart,
       isWishlistOpen: state => state.ui.wishlist
-
     })
   },
   methods: {
     onOrderConfirmation (payload) {
       this.loadOrderConfirmation = true
       this.ordersData = payload
       EventBus.$emit('modal-show', 'modal-order-confirmation')
+    },
+    fetchMenuData () {
```

Comment:
Shouldn’t we move: cart/load, wishlist/load and compare/load here as well to make the Woshlist.ts, Microcart.ts more “dummy”? It’s the same case like with categories

Reply:
Good idea. Can you make an issue for this? It shouldn't be part of this PR.